redash/query_runner/prometheus.py

```diff
@@ -37,6 +39,30 @@ def get_range_rows(metrics_data):
     return rows
 
 
+# Convert datetime string to timestamp
+def convert_query_range(payload):
+    range = {}
```

Comment:
range is a built-in function in Python, better use a different name.

Reply:
I resolved and pushed